realm_reference_docs_help.md

```diff
@@ -0,0 +1,36 @@
+# Realm Docs Team SDK Reference Manual (API Docs) Collaboration
+
+# Motivation
+
+- API documentation could use some love/attention
+- Docs folks can add an extra set of eyes to make sure that descriptions are current, language is clear, etc.
+- Wordings, sentence structure, descriptions
+
+# Product Description
+
+## Proposed Solution
+
+- docs writers help with API docs, updating them when they notice gaps or inconsistencies
+- tickets to improve API docs independently
+- docs writers as a highly suggested LGTM on SDK work (anywhere that touches code or logic that impacts API docs)
+    - help with wording suggestions
+    - help by asking questions, suggesting additional information to add
+
+## Goals
+
+- build comradery and relationships between docs and the realm SDK teams
+- improve quality of API docs
+- improve wording of API docs, make more consistent with docs docs
+
+## Non-Goals
+
+- docs team will not block SDK work -- we're just helping out, trying to make the API docs even better
+
+## Use Cases
+
+- all realm-<language> sdk repos
+
+# Success Metrics
+
+- API docs correctness and clarity is improved
```

Comment:
Would be great to have some measure for this even if it's something subjective like 9 out of 10 Realm SDK  engineers recommend the improved version.